lib/launcher.ts

```diff
@@ -251,7 +254,7 @@ let initFn = function(configFile: string, additionalConfig: Config) {
         let createNextTaskRunner = () => {
           let task = scheduler.nextTask();
           if (task) {
-            let taskRunner = new TaskRunner(configFile, additionalConfig, task, forkProcess);
```

Comment:
Great point! I don't think this will prevent options in`argv` from flowing through to the runners : `additionalConfig` is already [merged into config](https://github.com/angular/protractor/blob/master/lib/launcher.ts#L100-L102) and isn't referenced elsewhere in launcher. `multiCapabilities` related mutations happen directly to the `config` object.  Is the use case you are suggesting that `additionalConfig` may have options that we want to override modifications to to config that happen in between the initial parsing and (e.g. our detection of [`capabilities` or `multiCapabilities`](https://github.com/angular/protractor/blob/master/lib/launcher.ts#L143-L156))?

I wasn't able to find a test that verified that contract though :|. Do we have a pattern for testing cli and config related interactions?